openfe/tests/protocols/test_rfe_tokenization.py

```diff
@@ -39,7 +39,7 @@ def instance(self):
 
 class TestRelativeHybridTopologyProtocol(GufeTokenizableTestsMixin):
     cls = openmm_rfe.RelativeHybridTopologyProtocol
-    key = "RelativeHybridTopologyProtocol-bfb2f60bea0352a15bf6832d14d6e46e"
+    key = "RelativeHybridTopologyProtocol-99069cb606c80c9884a024c17fdab2e4"
```

Comment:
I know this PR has sat around for a while, but we don't track these keys anymore, right?

Reply:
Removed.